src/deps/tinyxml2/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+add_library(tinyxml2 STATIC
+  tinyxml2.h
+  tinyxml2.cpp
+)
+
+set_directory_properties(PROPERTIES CORRADE_USE_PEDANTIC_FLAGS ON)
```

Comment:
Missed this, it's causing a lot of `-Wzero-as-null-pointer-constant` warnings. I think we shouldn't enable pedantic flags for 3rd party dependencies, as we can't really control what's there. Opened #1233 to remove it.